content/engineering/incidents/playbooks/ci.md

```diff
@@ -0,0 +1,179 @@
+# CI Playbook
+
+> This page is maintained by the [DevX Team](../../engineering/enablement/dev-experience/index.md).
+
+> Audience: any software engineer, no prior infrastructure knowlegde required.
+
+This document sums up what to do in multiple scenarios that can block the CI.
+
+See also: [Continuous Integration](https://docs.sourcegraph.com/dev/background-information/continuous_integration) in our product documentation.
+
+## Prerequisites
+
+In order to handle problems with the CI:
+
+- Have access to the `sourcegraph-ci` _project_ on Google Cloud Platform.
+  - See [#it-tech-ops](https://sourcegraph.slack.com/archives/C01CSS3TC75)
+- Have the CLI `gcloud` tool installed and have authenticated yourself.
+  - See [Gain access to the cluster](../../engineering/deployments/debugging/tutorial/index.md)
+- Set the current Kubernetes context to the `default-buildkite` _cluster_ in the `sourcegraph-ci` _project_:
+  - `gcloud config set project sourcegraph-ci`
+  - `gcloud container clusters get-credentials default-buildkite --zone us-central1-c`
```

Comment:
I think this should be part of `Gain access to the cluster` (if it isn't already)

Reply:
That page does mention these instructions, just not for the CI cluster. I thought it would be useful to have this here, it makes this self-contained in some way. 

Still, I agree with you that it should be present on that page, I've added some content in there.